pyproject.toml

```diff
@@ -19,11 +19,19 @@ pyaml = "^21.10.1"
 oaklib = "^0.1.55"
 pyserde = "^0.9.7"
 docker = "^6.0.1"
+pheval = {git = "https://github.com/monarch-initiative/pheval.git"}
```

Comment:
We should probably not do this.. But if it is very convenient, then make an issue to change this to a normal system. @souzadevinicius - how do you release pheval to pypi?

Reply:
As there isn't a recent release of PhEval on pypi this is the only way I can get things to pass with the qc

Reply:
Can you request from Vinicius a workflow that makes a pypi release of pheval every time we merge to master? Does it make sense?